test/test_prototype_transforms_functional.py

```diff
@@ -1168,25 +1157,67 @@ def _compute_expected_bbox(bbox, padding_):
         torch.testing.assert_close(output_boxes, expected_bboxes)
 
 
+@pytest.mark.parametrize("device", cpu_and_gpu())
+def test_correctness_pad_segmentation_mask_on_fixed_input(device):
```

Comment:
Does this test have any value with the other test added?

Reply:
This test checks one option on cuda device instead of [cuda, cpu] x all_options from below test